cns/restserver/internalapi_windows.go

```diff
@@ -1,11 +1,55 @@
 package restserver
 
 import (
+	"context"
+	"fmt"
+
 	"github.com/Azure/azure-container-networking/cns"
 	"github.com/Azure/azure-container-networking/cns/types"
+	"github.com/Microsoft/hcsshim"
+	"github.com/pkg/errors"
 )
 
 // nolint
 func (service *HTTPRestService) programSNATRules(req *cns.CreateNetworkContainerRequest) (types.ResponseCode, string) {
 	return types.Success, ""
 }
+
+// setVFForAccelnetNICs is used in SWIFTV2 mode to set VF on accelnet nics
+func (service *HTTPRestService) setVFForAccelnetNICs() error {
+	// supply the primary MAC address to HNS api
+	macAddress, err := service.getPrimaryNICMACAddress()
+	if err != nil {
+		return err
+	}
+	macAddresses := []string{macAddress}
+	if _, err := hcsshim.SetNnvManagementMacAddresses(macAddresses); err != nil {
+		return errors.Wrap(err, "Failed to set primary NIC MAC address")
+	}
+	return nil
+}
+
+// getPrimaryNICMacAddress fetches the MAC address of the primary NIC on the node.
+func (service *HTTPRestService) getPrimaryNICMACAddress() (string, error) {
+	res, err := service.wscli.GetInterfaces(context.TODO())
```

Comment:
nit: instead of using `context.TODO()`, lets use a `context.WithTimeout()` here since we are exec-ing powershell command?